ledger/accountdb_test.go

```diff
@@ -785,3 +789,81 @@ func TestAccountsDbQueriesCreateClose(t *testing.T) {
 	qs.close()
 	require.Nil(t, qs.listCreatablesStmt)
 }
+
+func benchmarkWriteCatchpointStagingBalancesSub(b *testing.B) {
+	genesisInitState, _ := testGenerateInitState(b, protocol.ConsensusCurrentVersion)
+	const inMem = false
+	log := logging.TestingLog(b)
+	cfg := config.GetDefaultLocal()
+	cfg.Archival = false
+	log.SetLevel(logging.Warn)
+	dbBaseFileName := strings.Replace(b.Name(), "/", "_", -1)
+	l, err := OpenLedger(log, dbBaseFileName, inMem, genesisInitState, cfg)
+	require.NoError(b, err, "could not open ledger")
+	defer func() {
+		l.Close()
+		os.Remove(dbBaseFileName + ".block.sqlite")
+		os.Remove(dbBaseFileName + ".tracker.sqlite")
+	}()
+	catchpointAccessor := MakeCatchpointCatchupAccessor(l, log)
+	catchpointAccessor.ResetStagingBalances(context.Background(), true)
+	accountsCount := uint64(b.N)
+	b.ResetTimer()
```

Comment:
Practically it won't matter, but it's better to reset the timer right before the loop.